tests/slow/src/main/scala/tests/BaseMillServerSuite.scala

```diff
@@ -0,0 +1,37 @@
+package tests
+
+import scala.concurrent.ExecutionContext
+
+import scala.meta.internal.builds.ShellRunner
+import scala.meta.internal.metals.JdkSources
+import scala.meta.io.AbsolutePath
+
+trait BaseMillServerSuite {
+
+  def killServer(
+      workspace: AbsolutePath
+  )(implicit ec: ExecutionContext): Option[Option[String]] = {
+
+    val home = JdkSources.defaultJavaHome(None).head
+    ShellRunner
+      .runSync(
+        List(s"${home}/bin/jps", "-l"),
```

Comment:
This seems very linux specific. I know that we run that part of CI on linux only but I would be better if it worked for all systems for the sake of running tests locally.

Reply:
Och, I am not even sure how to kill on Windows :sweat: I can just add a guard maybe?

Reply:
And on Mac this should work the same

Reply:
Then add a guard, just so it doesn't throw errors.

Reply:
I added a condition on Windows. We can fix it if we ever need to run those tests on windows